### ISSUE TITLE
As simple as possible, JSON?

### ISSUE THREAD

**Issue by dkotrada** [activey/bob-builder#58]
> We try to keep Valder as simple and intuitive as possible mantaining it's elastic and quite powerful design.

Why don‘t you use for config files the Vala itself? Or .toml. https://en.m.wikipedia.org/wiki/TOML 
They have much cleaner syntax than json files. I mean easier to read and write for people than json.

**Comment by activey** [comment]
Hmm, honestly, I was a little bit more familiar with JSON when I was developing Valder, maybe it would make more sense to use something different ...

**Comment by dkotrada** [comment]
Nice webpage for valder. But my first impression was: "he write it is simple but the wirst example seems to compicated for me".  I think it is not attractive for first time visitors to your webpage. Take a look at this getting started page. It has some ideas to take for valder webpage. I mean step by step instead of complete config file. 
There are more alternatives: 
Human Json https://hjson.org or 
YAML https://en.m.wikipedia.org/wiki/YAML

**Comment by activey** [comment]
Yeah, already got some suggestions about config file format, I agree it could be simpler ... Just had no time for changing that ...

**Comment by naaando** [comment]
I thought the same, what if it were done format agnostic?

**Comment by activey** [comment]
Sure, go ahead and try it :)
